### PR TITLE
fix: avoid double thickness border

### PR DESCRIPTION
**Pull request by edoardo**
### Key features

1. do not use `border-collapse: separate` which causes the table border to have double thickness

---

### Description

`border-collapse: separate` was introduce to solve the issue of seeing the table content in the border space between cells when using fixed headers and scrolling.
The solution implemented in this PR is to manually add border rules for `th`, `td` and `table` to simulate a 1px border all around.
This solves the issue described above, but there is an issue, read below.

---

### Known issues

-   [x] When using fixed headers where the bottom/right border of the header disappears when scrolling.
Screenshot 2021-09-10 at 14 29 32

---

### Screenshots

Before:
Screenshot 2021-09-10 at 14 27 53

After:
Screenshot 2021-09-10 at 14 28 58

### CODE REVIEW

File: src/components/PivotTable/styles/PivotTable.style.js [hunk 1]
```diff
@@ -9,21 +9,54 @@ import {
...
         color: ${colors.grey900};
     }
+
     table {
         border-spacing: 0;
-        border-collapse: separate;
         white-space: nowrap;
         box-sizing: border-box;
         text-align: center;
+        border: 1px solid #b2b2b2;
```
Comment (martinkrulltott) [inline src/components/PivotTable/styles/PivotTable.style.js]: ui color instead of hex value?

Comment (martinkrulltott) [reply src/components/PivotTable/styles/PivotTable.style.js]: same for the border a few lines below

File: src/components/PivotTable/PivotTableContainer.js [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+                data-test="visualization-container"
+            >
+                <style jsx>{tableStyle}</style>
+                {width === 0 || height === 0 ? null : (
+                    <table
+                        className={classnames({
+                            'fixed-headers':
+                                engine.options.fixColumnHeaders &&
+                                engine.options.fixRowHeaders,
+                            'fix-column-headers':
```
Comment (martinkrulltott) [inline src/components/PivotTable/PivotTableContainer.js]: why both `fixed` and `fix`? use `fixed` everywhere?

```suggestion
                            'fixed-column-headers':
```

**Review by martinkrulltott — APPROVED** [review]
LGTM, but there are 2 very minor things that can be improved.

File: src/components/PivotTable/styles/PivotTable.style.js (hunk 1 above)
Comment (edoardo) [reply src/components/PivotTable/styles/PivotTable.style.js]: There is no such color in `ui`, but I added a `const` instead.

**Pull request merged**